File: src/main/kotlin/com/sg/launcher/repository/UserCredentialsRepository.java
package com.sg.launcher.repository;

import com.sg.launcher.repository.entity.UserCredentialsEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface UserCredentialsRepository extends JpaRepository<UserCredentialsEntity, Long> {
    UserCredentialsEntity findByUserId(Long userId );
}

File: src/main/kotlin/com/sg/launcher/repository/UserCredentialsRepository.kt
package com.sg.launcher.repository

import com.sg.launcher.repository.entity.UserCredentialsEntity
import org.springframework.data.jpa.repository.JpaRepository

interface UserCredentialsRepository : JpaRepository<UserCredentialsEntity?, Long?> {
    fun findByUserId(userId: Long?): UserCredentialsEntity?
}